includes/class-sensei-messages.php

```diff
@@ -157,7 +157,15 @@ public function send_message_link( $post_id = 0, $user_id = 0 ) {
 
 		}
 
-		echo $html;
+        /**
+         * Filter the send message link
+         *
+         * @since 1.9.17
+         *
+         * @param string  	$html
+         * @param array 	$this->message_notice
+         */
+        echo apply_filters( 'sensei_send_message_link', $html, $this->message_notice );
```

Comment:
could we also pass `$post_id` and `$user_id` (the method parameters) into the filter?